test/error-handler.test.js

```diff
@@ -9,59 +9,43 @@ var assert = require('assert');
 var request = require('supertest');
 
 describe('loopback.errorHandler(options)', function() {
-  it('should return default middleware when options object is not present', function(done) {
-    //arrange
-    var app = loopback();
-    app.use(loopback.urlNotFound());
-    app.use(loopback.errorHandler({ log: false }));
-
-    //act/assert
-    request(app)
-      .get('/url-does-not-exist')
-      .end(function(err, res) {
-        assert.ok(res.error.text.match(/<ul id="stacktrace"><li> &nbsp; &nbsp;at raiseUrlNotFoundError/));
-
-        done();
-      });
+  it('should pass strong-error-handler message to migrate', function(done) {
+    try {
+  //arrange
+      var app = loopback();
+      app.use(loopback.urlNotFound());
+      app.use(loopback.errorHandler({ log: false }));
```

Comment:
I didn't notice this comment was not addressed. I cleaned up the test in https://github.com/strongloop/loopback/commit/42c83f69a090664bb025d89f8dd5f5a7a3ffa34e